getting-started.adoc

```diff
@@ -1,3 +1,123 @@
+
+We're going to configure a Venmo account:
+
+.Configuring a Venmo account. In this case, we only need to edit the full name & username fields.
+image::configure-fiat-account.png[Configuring a national currency account]
+
+It's *crucial* the information you enter here is correct. Your trading partner can only acknowledge your payment if the they receive a payment from the same account as the account detailed here. Once you've created an account, you cannot edit its information (if you need to make a change, you can delete & add the account again).
+
+The "Account Name" field is an exception: it's just a label only you will ever see, so there's no right or wrong value for it. You can make it whatever you like.
+
+Once you're done, hit the "Save New Account" button and go back to the Buy BTC panel. Offers requiring the payment method you just configured should now be enabled:
+
+.After configuring a payment account: we picked Venmo, so Venmo offers are now enabled.
+image::after-adding-fiat-account.png[Configuring a national currency account]
+
+== Fund your Bisq wallet
+
+We're almost there—last thing: before you buy bitcoin with Bisq, you'll need some bitcoin.
+
+Huh?
+
+Here's why: to encourage fair trades, Bisq requires buyers and sellers to post a security deposit. On the buy side, it's {btc_deposit} BTC, along with a tiny bit more to cover the trading fee & mining fees for the transfers.
+
+[NOTE]
+.I'm new to this and have no bitcoin. How can I get bitcoin for the deposit?
+====
+You can buy bitcoin on a centralized exchange, but if you're interested in Bisq, chances are you might not want to do that.
+
+So what can you do? You've got options: friends, family, bitcoin ATMs, bitcoin meetups (commonly called "Satoshi Squares" but there are many others), vouchers, a site like LocalBitcoins.com, or your https://www.youtube.com/watch?v=GbmWGrQjgTA[friendly neighborhood airdrop^] (although you might need to time-travel to find one of these now!).
+====
+
+To find out exactly how much bitcoin you need to execute a specific trade, choose an offer you like. A new "Take Offer" tab will appear with some deal details. Click "Next step" at the bottom. Accept the disclaimer, and you should see a box that looks like this:
+
+<screenshot of box specifying total needed to fund trade>
+[[[caption: In this case, you'll need 0.012184 BTC to start the trade.]]]
+
+Once you've got the bitcoin you need, you can move it into Bisq's built-in wallet. This isn't required, but it'll make trading more convenient because Bisq could automatically transfer the funds it needs (since they'll be stored in a wallet Bisq controls). Otherwise, you'll need to transfer the funds into Bisq every time you start a trade. More on this below.
+
+Now that you've got enough bitcoin for a deposit, you're ready to trade!
+
+== Place a trade
```

Comment:
And really, it should be "Make an offer to trade" or "Take an offer to trade" to be consistent with our own language elsewhere, and to avoid introducing potentially confusing additional words for the same thing. 'Place' is basically synonymous with 'make', but in the steps that follow, what the user will in fact do is _take an offer_. So we should probably title the section "Take an offer". And I think this is fine, because that is the new user experience we want for people: they show up, they see attractive offers in their currency + payment method of choice, and they take one. It's this "happy path" that we're documenting, and you can of course do a sidebar below about "what if I don't find an offer I want to take?".